### ISSUE TITLE
Jira bot DMs you if you mention yourself in a comment on a Jira issue

### ISSUE THREAD

**Issue by mickmister** [mattermost/mattermost-plugin-jira#857]

**Comment by Kshitij-Katiyar** [comment]
The issue you mentioned is not reproducible on my Mattermost server version 8.0.1.
Also, this case is alreay handled in the code, please check the below link for the same
https://github.com/mattermost/mattermost-plugin-jira/blob/97184a537109367fa455bbe6700f59f0950211d0/server/webhook_parser.go#L273C4-L273C4

**Comment by mickmister** [comment]
> Also, this case is already handled in the code, please check the below link for the same

That code was added 4 years ago, so I would expect that to be applied to the version I reported this on, in May 2022. Maybe there's a bug where that if statement is not correct.

https://github.com/mattermost/mattermost-plugin-jira/blob/97184a537109367fa455bbe6700f59f0950211d0/server/webhook_parser.go#L273-L275

> The issue you mentioned is not reproducible on my Mattermost server version 8.0.1.

The plugin version and Jira instance type / version is really what matters here. Things change on Jira's side from time to time, including how users are referenced in comments. In the case of me reporting this, I'm thinking I experienced this on Jira Cloud in production.
